phase0/CRC.md

```diff
@@ -0,0 +1,205 @@
+---
+marp: true
+title: Marp CLI example
+description: Hosting Marp slide deck on the web
+theme: gaia
+---
+
+<style>
+:root {
+  font-size: 24px;
+}
+</style>
+
+<!-- _class: lead -->
+# Entity Classes
+
+---
+
+# `abstract class IndividualCharacter`
+
+### Responsibilities
+* Health: Attribute, int, denotes the total amount of health the individual has
+* Position: Attribute, array with x, y, where on the map the IC is currently located
+* Inventory: Attribute, ArrayList, items currently being held
+* Level(?): Attribute, the level of the enemy
+* Range(?): Attribute, how far the character can attack
+* Hitbox(?): Attribute, from how far the character can be hit from
+* updatePosition: Method, updates the x, y position coordinates
+* depleteHealth: Method, updates health after being attacked 
+
+### Collaborators
+Player, Enemy, Defender
+
+---
+
+# `class Player`
+
+### Responsibilities
+* Spawnable_defenders: Attribute, dictionary of level of defenders and their corresponding integer of number of defenders the player can spawn.  
+* Build_wall(position): Method to use inventory to build wall at a certain position on the map
+* Spawn_defender(position): Method to spawn defender at a certain position on the map
```

Comment:
could be replaced by a "use item" method if defenders and walls were inventory items

Reply:
but also entity classes do not have mutators

Reply:
all agreed; make this change